diff --git a/Libraries/Components/Keyboard/KeyboardAvoidingView.js b/Libraries/Components/Keyboard/KeyboardAvoidingView.js
index 6eca1569f3024881ffd645aa026dc5ee4090d93b..1716d959828c66aac3c6387e9f4e10cf01711b51 100644
--- a/Libraries/Components/Keyboard/KeyboardAvoidingView.js
+++ b/Libraries/Components/Keyboard/KeyboardAvoidingView.js
@@ -8,23 +8,22 @@
  * @format
  */
 
-import type {ViewStyleProp} from '../../StyleSheet/StyleSheet';
+import type { ViewStyleProp } from '../../StyleSheet/StyleSheet';
 import type {
-  ViewLayout,
-  ViewLayoutEvent,
-  ViewProps,
+    ViewLayout,
+    ViewLayoutEvent
 } from '../View/ViewPropTypes';
-import type {KeyboardEvent, KeyboardMetrics} from './Keyboard';
+import type { KeyboardEvent, KeyboardMetrics } from './Keyboard';
 
+import * as React from 'react';
+import { createRef } from 'react';
 import LayoutAnimation from '../../LayoutAnimation/LayoutAnimation';
 import StyleSheet from '../../StyleSheet/StyleSheet';
 import Platform from '../../Utilities/Platform';
-import {type EventSubscription} from '../../vendor/emitter/EventEmitter';
+import { type EventSubscription } from '../../vendor/emitter/EventEmitter';
 import AccessibilityInfo from '../AccessibilityInfo/AccessibilityInfo';
 import View from '../View/View';
 import Keyboard from './Keyboard';
-import * as React from 'react';
-import {createRef} from 'react';
 
 export type KeyboardAvoidingViewProps = $ReadOnly<{
   ...ViewProps,
@@ -208,7 +207,7 @@ class KeyboardAvoidingView extends React.Component<
       ];
     } else {
       this._subscriptions = [
-        Keyboard.addListener('keyboardDidHide', this._onKeyboardChange),
+        Keyboard.addListener('keyboardDidHide', this._onKeyboardHide),
         Keyboard.addListener('keyboardDidShow', this._onKeyboardChange),
       ];
     }
